code/game/world.dm

```diff
@@ -164,6 +167,14 @@ GLOBAL_VAR(restart_counter)
 	log_runtime(GLOB.revdata.get_log_message())
 
 /world/Topic(T, addr, master, key)
+	if(!SSfail2topic)
+		return "Server not initialized."

```

Comment:
This will either 
a) Never be hit (since I think it's impossible for SS's to not exist before topics can be sent)
b) Interfere with TGS calls

I'm not a fan of adding it

Reply:
It's possible to send topic calls during the very early stages of server init. This is mainly just to make output verbose where there would otherwise just be null output as a result of runtimes.